client/components/footer/footer.controller.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+angular.module('observatory3App')
+  .controller('MainCtrl', function ($scope, $http) {
```

Comment:
This shouldn't be called MainCtrl. And doesn't look like a controller is even needed at this point. You could leave it out for now. We can add it back when we need more logic/functionality